condor_watch_q.py

```diff
@@ -432,16 +436,18 @@ def table_by_event_log(self, abbreviate_path_components=False):
                     if len(live_proc_ids) == 1:
                         x = "{}.{}".format(cluster_id, live_proc_ids[0])
                     else:
-                        x = "{}.{}-{}".format(
-                            cluster_id, min(live_proc_ids), max(live_proc_ids)
+                        x = "{}.{}".format(
```

Comment:
Not sure what the intent is here, but the changed code is broken (format call with wrong number of `{}` in the string).